shared/src/commonMain/kotlin/org/hyperskill/app/step_quiz_code_blanks/presentation/StepQuizCodeBlanksFeature.kt

```diff
@@ -0,0 +1,56 @@
+package org.hyperskill.app.step_quiz_code_blanks.presentation
+
+import org.hyperskill.app.analytic.domain.model.AnalyticEvent
+import org.hyperskill.app.step.domain.model.Step
+import org.hyperskill.app.step_quiz_code_blanks.domain.model.CodeBlock
+import org.hyperskill.app.step_quiz_code_blanks.domain.model.Suggestion
+import org.hyperskill.app.step_quiz_code_blanks.view.model.StepQuizCodeBlanksViewState
+
+object StepQuizCodeBlanksFeature {
+    internal fun isCodeBlanksFeatureAvailable(step: Step): Boolean =
+        step.block.options.codeBlanksStrings.isNullOrEmpty().not()
+
+    internal fun initialState(): State = State.Idle
+
+    sealed interface State {
+        data object Idle : State
+
+        data class Content(
+            val step: Step,
+            val codeBlocks: List<CodeBlock>
+        ) : State {
+            internal val codeBlanksStringsSuggestions: List<Suggestion.ConstantString> =
+                step.block.options.codeBlanksStrings?.map { codeBlanksString ->
+                    buildString {
+                        if (!codeBlanksString.startsWith("\"")) {
+                            append("\"")
+                        }
+                        append(codeBlanksString)
+                        if (!codeBlanksString.endsWith("\"")) {
+                            append("\"")
+                        }
```

Comment:
For now it's ok to add quotes, but in future we'll support strings with formatting and it'll be content's responsibility to configure code blanks strings properly. I fixed options that don't have quotes, so we can remove this code now